### PR TITLE
Html css popup

### PR DESCRIPTION
**Pull request by Zakkarat**
### html-css-popup

[Demo](https://zakkarat.github.io/popup-menu/) |
[Code base](https://github.com/Zakkarat/popup-menu)

The code is submitted in its own sub-directory and in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/Zakkarat/html-css-popup/index.html [hunk 1]
```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
```
Comment (A-Ostrovnyy) [inline submissions/Zakkarat/html-css-popup/index.html]: Add meta tag for mobile devices.

File: submissions/Zakkarat/html-css-popup/index.html [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+    <link rel="stylesheet" href="./css/main.css">
+    <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
+    <title>Popup</title>
+  </head>
+  <body>
+        <div class="main">
+          <a href="#">LMail</a>
+          <a href="#">Images</a>
+          <input type="checkbox" id ="menu">
+          <label for="menu"><img src="./images/icons.png"></label>
```
Comment (A-Ostrovnyy) [inline submissions/Zakkarat/html-css-popup/index.html]: Don't forget fill `alt` attribute to the `img` tag.

File: submissions/Zakkarat/html-css-popup/index.html [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+            </div>
+            <div class="card">
+              <img src="./images/logo_sheets_128px.png" />
+              Sheets
+            </div>
+            <div class="card">
+              <img src="./images/logo_admin_128px.png" />
+              Settings
+            </div>
+            <br class="hidden">
```
Comment (A-Ostrovnyy) [inline submissions/Zakkarat/html-css-popup/index.html]: This tag was designed not for that.

File: submissions/Zakkarat/html-css-popup/css/main.css [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+  align-self: flex-end;
+  align-items: center;
+  justify-content: center;
+  background-color: #e8e8e8;
+}
+.card > img {
+  width: 64px;
+  height: 64px;
+}
+input {
```
Comment (A-Ostrovnyy) [inline submissions/Zakkarat/html-css-popup/css/main.css]: Avoid styling by tag.
Separate responsibilities: HTML tag - markup, CSS Classes - styling, `#id` - Javascript(logic).

File: submissions/Zakkarat/html-css-popup/index.html (hunk 1 above)
Comment (Zakkarat) [reply submissions/Zakkarat/html-css-popup/index.html]: Forgot about it, now it's there!

File: submissions/Zakkarat/html-css-popup/index.html (hunk 2 above)
Comment (Zakkarat) [reply submissions/Zakkarat/html-css-popup/index.html]: Done!

File: submissions/Zakkarat/html-css-popup/index.html (hunk 3 above)
Comment (Zakkarat) [reply submissions/Zakkarat/html-css-popup/index.html]: Forgot to delete it. Now it's done.

File: submissions/Zakkarat/html-css-popup/css/main.css (hunk 4 above)
Comment (Zakkarat) [reply submissions/Zakkarat/html-css-popup/css/main.css]: Done!

File: submissions/Zakkarat/html-css-popup/css/main.css [hunk 5]
```diff
@@ -97,7 +95,9 @@ a:hover {
   cursor: pointer;
 }
 input {
```
Comment (A-Ostrovnyy) [inline submissions/Zakkarat/html-css-popup/css/main.css]: Avoid styling by tag.

Comment (Zakkarat) [reply submissions/Zakkarat/html-css-popup/css/main.css]: Now its fully done!

File: submissions/Zakkarat/html-css-popup/index.html [hunk 6]
```diff
@@ -12,92 +12,92 @@
           <a href="#" class="refer">LMail</a>
           <a href="#" class="refer">Images</a>
           <input class="hideBoxes hide boxes" type="checkbox" id ="menu">
-          <label class="icons" for="menu"><img src="./images/icons.png"></label>
+          <label class="icons" for="menu"><img src="./images/icons.png" tabindex="0"></label>
```
Comment (A-Ostrovnyy) [inline submissions/Zakkarat/html-css-popup/index.html]: Remove `tabindex="0"` and use 
```
.boxes:focus + .icons {
    outline: auto;
}
```
Same story with button More.

File: submissions/Zakkarat/html-css-popup/index.html [hunk 7]
```diff
@@ -12,92 +12,92 @@
...
           <div class="menu">
             <div class="container">
-            <div class="card">
+            <div class="card" tabindex="0" tabindex="0">
```
Comment (A-Ostrovnyy) [inline submissions/Zakkarat/html-css-popup/index.html]: Remove duplicated property `tabindex`.

Comment (Zakkarat) [reply submissions/Zakkarat/html-css-popup/index.html]: Done!

File: submissions/Zakkarat/html-css-popup/index.html (hunk 6 above)
Comment (Zakkarat) [reply submissions/Zakkarat/html-css-popup/index.html]: Thank you for advice! Done.

**Comment by A-Ostrovnyy** [comment]
Well done!

**Pull request merged**